### PR TITLE
10320 migrate jean pants to formation

### PR DESCRIPTION
**Pull request by rianfowler**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -207,7 +207,7 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/jean-pants": "^0.6.1",
+    "@department-of-veterans-affairs/formation": "^0.7.0",
```
Comment (rianfowler) [inline package.json]: updated dependency name

File: src/js/auth/containers/AuthApp.jsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { connect } from 'react-redux';
 
 import recordEvent from '../../../platform/monitoring/record-event';
-import LoadingIndicator from '@department-of-veterans-affairs/jean-pants/LoadingIndicator';
+import LoadingIndicator from '@department-of-veterans-affairs/formation/LoadingIndicator';
```
Comment (rianfowler) [inline src/js/auth/containers/AuthApp.jsx]: most changes are just this- changing the directory to `formation`

File: yarn.lock [hunk 3]
```diff
@@ -2,9 +2,9 @@
 # yarn lockfile v1
 
 
-"@department-of-veterans-affairs/jean-pants@^0.6.1":
-  version "0.6.1"
-  resolved "https://registry.yarnpkg.com/@department-of-veterans-affairs/jean-pants/-/jean-pants-0.6.1.tgz#17c8ad92fe7c9cdef37e50f04bab43dbba5363b8"
+"@department-of-veterans-affairs/formation@^0.7.0":
```
Comment (rianfowler) [inline yarn.lock]: yarn.lock updated

File: src/platform/site-wide/user-nav/sass/user-nav.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
-@import "~@department-of-veterans-affairs/jean-pants/sass/shared-variables";
-@import "~@department-of-veterans-affairs/jean-pants/sass/modules/m-modal";
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
```
Comment (rianfowler) [inline src/platform/site-wide/user-nav/sass/user-nav.scss]: includes updates to sass files

**Comment by jbalboni** [comment]
@rianfowler Can you remove the package-lock.json file and add it to .gitignore?

**Review by jbalboni — APPROVED** [review]

**Pull request merged**